README.md

```diff
@@ -122,27 +122,28 @@ https://github.com/openebs/zfs-localpv/blob/HEAD/docs/faq.md#6-how-to-add-custom
 In order to support moving data to a new node later on, you must label each node with a unique value for `openebs.io/nodeid`.
 For more information on migrating data, please [see here](docs/faq.md#8-how-to-migrate-pvs-to-the-new-node-in-case-old-node-is-not-accessible)
 
+**NOTE:** Installation using operator YAMLs is not the supported way any longer.  
```

Comment:
@avishnu @tiagolobocastro Should we also mention here that, since this is not supported, there will be no upgrade path for someone who uses this way just to be very specific about our scopes.

Also, we have not updated the upgrade docs in this repo. The upgrade in this repo is all operator specific. How do we want to put upgrade paths here? We only have umbrella 3.10.0 to 4.0.0 upgrade path.